config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleMetadataBuilder.java

```diff
@@ -61,6 +62,7 @@ public BundleMetadata build(final AnnotatedBundle annotatedBundle, final Bundle
             builder.environmentEntities(getEnvironmentDependenciesMetadata(dependentEntities));
             builder.dependencies(annotatedBundle.getDependentBundles());
             builder.tags(annotatedEntity.getTags());
+            builder.portalTemplate(Boolean.valueOf(encass.getProperties().get(PORTAL_TEMPLATE).toString()));
```

Comment:
will this property be always there ? what about 9.4 and older versions ? should we have null check ?